### PR TITLE
deepac v0.9.1

### PR DESCRIPTION
**Pull request by JakubBartoszewicz**
Dear @bioconda/core Team,

Could I ask for a review? This is my first submission and a new package.

Thank you!

:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/deepac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - deepac
+    - deepac.eval
+  commands:
+    - deepac --help
+
+about:
+  home: https://gitlab.com/rki_bioinformatics/DeePaC
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (bgruening) [inline recipes/deepac/meta.yaml]: LICENSE

File: recipes/deepac/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  commands:
+    - deepac --help
+
+about:
+  home: https://gitlab.com/rki_bioinformatics/DeePaC
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: Predicting pathogenic potentials of novel DNA with reverse-complement neural networks.
+  doc_url:
```
Comment (bgruening) [inline recipes/deepac/meta.yaml]: please fill these out as well or remove, the dev_url

https://gitlab.com/rki_bioinformatics/DeePaC

File: recipes/deepac/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - deepac=deepac.command_line:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - biopython
```
Comment (bgruening) [inline recipes/deepac/meta.yaml]: I think all of those can be removed, but python and pip

**Comment by JakubBartoszewicz** [comment]
Thanks, done!

**Comment by bgruening** [comment]
Thanks @JakubBartoszewicz!

**Pull request merged**